packages/material-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -1,35 +1,39 @@
-import React from 'react';
+import React from "react";
 
-import { FieldTemplateProps } from '@rjsf/core';
+import { FieldTemplateProps } from "@rjsf/core";
 
-import FormControl from '@material-ui/core/FormControl';
-import FormHelperText from '@material-ui/core/FormHelperText';
-import List from '@material-ui/core/List';
-import ListItem from '@material-ui/core/ListItem';
-import Typography from '@material-ui/core/Typography';
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import Typography from "@material-ui/core/Typography";
 
 const FieldTemplate = ({
   id,
   children,
   displayLabel,
+  required,
   rawErrors = [],
   rawHelp,
   rawDescription,
 }: FieldTemplateProps) => {
   return (
-    <FormControl fullWidth={true} error={rawErrors.length ? true : false}>
+    <FormControl
+      fullWidth={true}
+      error={rawErrors.length ? true : false}
+      required={required}>
       {children}
       {displayLabel && rawDescription ? (
         <Typography variant="caption" color="textSecondary">
           {rawDescription}
         </Typography>
       ) : null}
       {rawErrors.length > 0 && (
-        <List dense={true}>
+        <List dense={true} disablePadding={true}>
```

Comment:
Why set disablePadding and disableGutters to true?

Reply:
I was trying to visually match the visual behavior here: https://material-ui.com/components/text-fields/#validation

With variants 'filled' and 'outlined' there's an indentation on the error display, but otherwise it's anchored to the edge with the input.